frontend/src/views/Collection/index.tsx

```diff
@@ -118,6 +121,16 @@ const Collection: FC = () => {
   const shouldShowPrivateWriteAction =
     collection.access_type === ACCESS_TYPE.WRITE && isPrivate;
 
+  const showTombstonedDatasetToast = () => {
+    Toast.show({
+      icon: IconNames.ISSUE,
+      intent: Intent.PRIMARY,
+      message:
+        `A dataset was withdrawn by ${collection.contact_name}. You've been redirected to the parent collection.`,
+    });
+    setHasTombestonedToastBeenShown(true)
```

Comment:
It seems like this is what's causing this error:

```
Warning: Cannot update during an existing state transition (such as within `render`). Render methods should be a pure function of props and state.
```

Reply:
Upon checking, it seems like the issue is coming from `Toast.show`. Without adding any other lines of code, I added the `Toast.show` snippet to the body of the component, and the error shows up.